content/interfaces/sql/sql-support/supported-statements.md

```diff
@@ -44,21 +44,23 @@ title: Supported Statements
 | `ALTER TYPE` | X | Column type changes are not supported. |
 | `ALTER VIEW` | X | Views can be created and dropped, but not altered. |
 | `CHANGE COLUMN` | ✓ |  |
-| `CREATE DATABASE` | X | Create new repositories with `dolt clone` or `dolt init` |
+| `CREATE DATABASE` | ✓ | Create an in memory database the length of a session. |
```

Comment:
in-memory

Reply:
Mention that creating new dolt databases that persist outside of a session is not yet supported.

Reply:
And give it a O instead of a check, for this reason